Command /bin/sh emitted errors but did not return a nonzero exit code to indicate failure

Originally reported in issue #200, seems like it is happening again in 0.7.1?

.../TQEntityCell.swift:226:67: warning: Force Unwrapping Violation: Force unwrapping should be avoided. (force_unwrapping)
.../TQEntityCell.swift:238:64: warLinting 'TQQueryCell.swift' (365/697)

--- a/Source/SwiftLintFramework/Extensions/QueuedPrint.swift
+++ b/Source/SwiftLintFramework/Extensions/QueuedPrint.swift
@@ -37,6 +37,7 @@ A thread-safe, newline-terminated version of fputs(..., stderr).
 */
 public func queuedPrintError(string: String) {
     dispatch_async(outputQueue) {
+        fflush(stdout)
         fputs(string + "\n", stderr)
     }
 }

